aeon/regression/sklearn/_rotation_forest_regressor.py

```diff
@@ -104,12 +104,12 @@ def __init__(
         min_group: int = 3,
         max_group: int = 3,
         remove_proportion: float = 0.5,
-        base_estimator: Optional[Type[BaseEstimator]] = None,
+        base_estimator: Optional[type[BaseEstimator]] = None,
```

Comment:
```suggestion
        base_estimator: Optional[BaseEstimator] = None,
```
and other locations in this file